packages/web-components/src/stories/tree-node.stories.mdx

```diff
@@ -51,12 +55,24 @@ export const Default = (args) => {
 
 export const Selected = (args) => {
     return html`
-<rux-tree-node ?selected="${args.selected}"> Level 1 </rux-tree-node>
+<rux-tree-node
+    ?selected="${args.selected}"
+    ?expanded="${args.expanded}"
+>
+    Level 1
+    <rux-tree-node slot="node"></rux-tree-node>
```

Comment:
What was the intention here adding the child slot? It might be better to just disable controls for these variants entirely because the Tree Nodes expand functionality won't work in isolation. They need to be inside a Tree component. Thinking about loud here, maybe it'd be better to have them under Tree then? similar to pop up menu items, etc.

Reply:
This was an error when copying and pasting, thanks.